src/options.ml

```diff
@@ -398,7 +398,24 @@ let init () =
   dir_reorder my_include_dirs include_dirs;
   dir_reorder my_exclude_dirs exclude_dirs;
 
-  ignore_list := List.map String.capitalize_ascii !ignore_list
+  ignore_list := List.map String.capitalize_ascii !ignore_list;
+
+  let raw_ocamlc_config = Command.run_and_read (S [!ocamlc ; A "-config" ]) in
+  let ocamlc_config_lines = String.split_on_char '\n' raw_ocamlc_config in
+  let ocamlc_configs =
+    List.concat_map
+      (fun config ->
+         match String.index_opt config ':' with
+         | Some sep -> [String.before config sep, String.after config (sep + 2)]
+         | None -> [])
```

Comment:
it's a micro-optimization but concat_map seemed overkill compared to a simple filter_map
```suggestion
    List.filter_map
      (fun config ->
         match String.index_opt config ':' with
         | Some sep -> Some (String.before config sep, String.after config (sep + 2))
         | None -> None)
```